### PR TITLE
Calls to sntp adjusted to the syntax of ntp 4.2.8 (bnc#937837)

### PR DESCRIPTION
**Pull request by ancorgs**

### CODE REVIEW

File: src/clients/ntp-client_proposal.rb [hunk 1]
```diff
@@ -399,12 +399,13 @@ def Write(param)
 
         Builtins.y2milestone("Running sntp to sync with %1", ntp_server)
 
-        # -s: do set the system time
+        # -S: do set the system time
```
Comment (jreidinger) [inline src/clients/ntp-client_proposal.rb]: np: why not simple `set the system time`

File: src/clients/ntp-client_proposal.rb [hunk 2]
```diff
@@ -399,12 +399,13 @@ def Write(param)
...
         # -t 5: timeout of 5 seconds
         # -l <file>: log to a file to not mess text mode installation
+        # -c: causes all IP addresses to which ntp_server resolves to be queried in parallel
```
Comment (jreidinger) [inline src/clients/ntp-client_proposal.rb]: np: maybe simple `querry all ip addresses in parallel`

**Comment by jreidinger** [comment]
LGTM

**Comment by ancorgs** [comment]
I took the descriptions for the comments directly from the bug reporter, who turns out to be the ntp maintainer. I think they are good enough.

**Pull request merged**